migrations/versions/48ee74b8a7c8_.py

```diff
@@ -61,10 +61,22 @@ class Token(Base):
                        default=u'', index=True)
 
 
+# Check if the SQL dialect uses sequences
+# (from https://stackoverflow.com/a/17196812/7036742)
+def dialect_supports_sequences():
+    return op._proxy.migration_context.dialect.supports_sequences
+
+
+def create_seq(name):
+    if dialect_supports_sequences():
+        op.execute(CreateSequence(Sequence(name)))
+
+
 def upgrade():
     try:
+        create_seq('tokenowner_seq')
```

Comment:
Looks good! But could you wrap this call in a try-except block? This way, if creating the sequence fails for any reason, we will at least create the tokenowner table.

Reply:
fixed in beb31afb